share/spice/playing_cards/playing_cards.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_playing_cards = function(api_result){
+        if(api_result.error || !api_result.cards) {
+            return Spice.failed('playing_cards');
+        }
+        
+        Spice.add({
+            id: "playing_cards",
+            name: "Deck of Cards",
+            data: api_result.cards,
+            meta: {
+                sourceName: "DeckOfCards",
+                sourceUrl: 'http://deckofcardsapi.com'
+            },
+            normalize: function(item) {
+                return {
+                    image: item.image
+                };
+            },
+            templates: {
+                group: 'movies',
```

Comment:
I think the problem is related to the padding on the `tile__body` which is below the `<img>` in the DOM. Take a peak and you should see it has a ton of padding. Set it to `0` and it should work.

If that doesn't help, then please push the code so I can run it to see what's going on